### DISCUSSION TITLE
Scroll by particular offset and take screenshots

### DISCUSSION THREAD

**Discussion by MokoPolo** [webdriverio/webdriverio#8935] [Q&A]
Hello,

Is it possible to scroll the page by a particular value and then each time take a screenshot? I thought broswer.scroll() might do it but seems to be deprecated

Thanks!

**Comment by christian-bromann** [comment]
You can use the [`performAction`](https://webdriver.io/docs/api/webdriver/#performactions) to do this, in `v8` we will introduce a `browser.scroll` command, see https://v8.webdriver.io/.

**Reply by MokoPolo** [reply]
Do you happen to have a sample on how to scroll? I'm kind of interested in finding how many times I need to scroll down a page and each time take a screenshot. It's kind of like browser.checkFullPageScreen() but each time I page down I might have to perform some additional logic

**Reply by christian-bromann** [reply]
I don't have an example at hand

**Reply by MokoPolo** [reply]
Do you have any place where I can ask about this type of thing? I'm new to webdriverio sorry

**Reply by christian-bromann** [reply]
Check out the docs: https://webdriver.io/community/support

**Reply by MokoPolo** [reply]
Thank you so much! I ended up using 

window.scrollBy(0, 500);

And it does what I need thanks!
